editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1278,7 +1278,7 @@ export const MetadataUtils = {
   },
   isComponentInstance(path: ElementPath, rootElements: Array<UtopiaJSXComponent>): boolean {
     const elementName = MetadataUtils.getStaticElementName(path, rootElements)
-    return elementName != null && !isIntrinsicElement(elementName)
+    return elementName != null && !isIntrinsicHTMLElement(elementName)
```

Comment:
I think this is fine but I wonder if we're missing anything else specific to non-HTML intrinsic elements :thinking:

Reply:
I combed over the code and pushing a commit where I fix a few more places that were confusing the two